flocker/node/test/test_script.py

```diff
@@ -433,15 +480,18 @@ def test_default_openstack(self):
         """
         An OpenStack backend is available by default.
         """
+        mimic_port = mimic_for_test(test_case=self)
```

Comment:
The problem here is that the Cinder backend driver factory now calls ``VolumeManager.list`` in order to detect whether the v1 or v2 API is available. 
These are unit tests though, so they really shouldn't depend on having responsive Cinder endpoints.

However, I found that mimic now has a Cinder v2 simple list endpoint which is just enough to get this test passing. 
It seems OTT for such a basic test....but it works.

Alternatives:
 * Remove the test.
 * Monkey patch ``cinderclient.v2.volumes.VolumeManager.list`` for this test.
 * Implement a lazy loading ``ICinderClient`` and alter ``cinder_from_configuration`` to use it. The Cinder API version will be automatically detected on first use.